com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -186,7 +186,6 @@ public bool IsCameraProjectionMatrixFlipped()
     [MovedFrom("UnityEngine.Rendering.LWRP")] public struct ShadowData
     {
         public bool supportsMainLightShadows;
-        public bool requiresScreenSpaceShadowResolve;
```

Comment:
Make obsolete instead of removing. We have to give users at least a package to have a change to upgrade.

Reply:
This should have been done before but I guess we missed it.

Reply:
Let me restore it and add 'obsolete' attribute